### PR TITLE
cmd/snap: unhide --name parameter to snap install, tweak help message

### PR DESCRIPTION
**Pull request by bboozzoo**
Unhide --name option. Tweak help message to mention instance key.

### CODE REVIEW

File: cmd/snap/cmd_snap_op.go [hunk 1]
```diff
@@ -45,7 +45,8 @@ var (
 )
 
 var longInstallHelp = i18n.G(`
-The install command installs the named snaps in the system.
+The install command installs the named snaps in the system under a given
+instance name.
```
Comment (chipaca) [inline cmd/snap/cmd_snap_op.go]: paging @degville

Comment (degville) [reply cmd/snap/cmd_snap_op.go]: I think I'd make the first line simple, ie. `The install command installs the named snaps on the system.`

And add something like the following on a second line:
`To install multiple instances of the same snap, append an underscore and a unique identifier (for each instance) to a snap's name.`

**Comment by pedronis** [comment]
I wonder if doing this makes sense if they are still behind the flag anyway or add to the confusion.

File: cmd/snap/cmd_snap_op.go [hunk 2]
```diff
@@ -923,7 +929,7 @@ func init() {
 			// TRANSLATORS: This should not start with a lowercase letter.
 			"unaliased": i18n.G("Install the given snap without enabling its automatic aliases"),
 			// TRANSLATORS: This should not start with a lowercase letter.
-			"name": i18n.G("Install the snap file under given snap name"),
+			"name": i18n.G("Install the snap file under given instance name"),
```
Comment (degville) [inline cmd/snap/cmd_snap_op.go]: Maybe add *the*: Install the snap file under *the* given instance name

**Comment by chipaca** [comment]
poke

**Pull request merged**